src/app.ts

```diff
@@ -87,7 +90,16 @@ if (process.env.NODE_ENV === 'production') {
       }
     },
     (req, res) => {
-      res.end('hello world')
+      try {
+        const mappings = getMappings()
+        const { ip, port } =
+          mappings.find(({ subDomain, domain }) => {
+            return `${subDomain}.${domain}` === req.headers.host
+          }) || {}
+        if (port) return proxy.web(req, res, { target: `${ip}:${port}` })
```

Comment:
Please handle errors gracefully.
```
if (!port) return res.send('Cannot redirect')
return proxy.web....
```

Reply:
and then please add error handler. When there is an error, please do `res.send('Error communicating with server that runs <hostname>')`